core/domain/html_cleaner.py

```diff
@@ -400,7 +400,7 @@ def validate_rte_tags(
 
         start_value = float(tag['start-with-value'].strip())
         end_value = float(tag['end-with-value'].strip())
-        if start_value > end_value and start_value != 0.0 and end_value != 0.0:
+        if start_value >= end_value and start_value != 0.0 and end_value != 0.0:
```

Comment:
Should we allow videos where `startTime === endTime`?

Reply:
@Lawful2002 In my opinion no since it does not make any sense to have it set to the same. Since the frontend requires start time > end time, the = sign was added to ensure consistency as discussed in the [issue](https://github.com/oppia/oppia/issues/18211)